stock_lot_suffix/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+This module can add a suffix to the stock_lot. The display of the name in the stock_lot changes by this module when a specific partner or analytic plan is used.
```

Comment:
Please use the `stock.lot` nomenclature (instead of `stock_lot`) for model technical names.